logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/selfdeletingMessages/SelfDeletionTimer.kt

```diff
@@ -46,6 +47,16 @@ sealed class SelfDeletionTimer {
         else -> Duration.ZERO
     }
 
+    private fun toLogMap(): Map<String, Any?> = mapOf(
+        typeKey to "SelfDeletionTimer",
+        selfDeletionKey to this::class.simpleName,
```

Comment:
not sure this will work with `simpleName`. it will show mangled on release builds. Please use a string instead